Roadmap.md

```diff
@@ -0,0 +1,44 @@
+# karmada operator roadmap
+
+The karmada operator is a new project of karmada community designed to manage the lifecycle of karmada. The following is the core feature of the karmada operator that we will work on. It is not a detailed to-do list, but represents overall development direction. It is hoped that more contributors will have a general understanding of project and participate in it.
+
+## 2022 H2
+- the `pr` and `issue` template
```

Comment:
I think roadmap is more like a plan for new features about karmada-operator. We should not add some the content of community governance.

Reply:
That's reasonable, but it’s every concise if we only list features of karmada, and the e2e and some CI actions is important part of jobs for us,  so I separate two part content to feature and others. so we can easy focus on features.

Reply:
Maybe we can talk about it on today‘s community meeting.

Reply:
ok